src/layout/Footer.tsx

```diff
@@ -15,39 +18,283 @@ export const Footer = () => {
         padding: '1rem',
         bgcolor: 'info.light',
         display: 'grid',
-        gridTemplateColumns: { xs: '1fr', lg: 'auto auto auto auto' },
-        gap: '0.5rem 1rem',
-        alignItems: 'center',
+        gridTemplateColumns: { xs: '1fr', lg: '1fr 1fr 1fr' },
+        gap: '3rem',
+        color: 'primary.main',
       }}>
-      <MuiLink
-        data-testid={dataTestId.footer.aboutLink}
-        component={Link}
-        to={UrlPathTemplate.About}
-        sx={{ justifySelf: { xs: 'center', lg: 'start' }, gridRow: { xs: 3, lg: 1 } }}>
-        {t('about.about_nva')}
-      </MuiLink>
-      <Typography sx={{ color: 'primary.main', gridColumn: { xs: 1, lg: 3 }, justifySelf: 'center' }}>
-        {t('about.footer_text')}
-      </Typography>
-      <LanguageSelector />
-      <Box
-        sx={{
-          display: 'flex',
-          justifySelf: { xs: 'center', lg: 'end' },
-          gap: '0.5rem',
-          gridColumn: { xs: 1, lg: 4 },
-        }}>
-        <MuiLink
-          data-testid={dataTestId.footer.availabilityStatement}
-          target="_blank"
-          rel="noopener noreferrer"
-          href={'https://uustatus.no/nb/erklaringer/publisert/bffb4b1d-25eb-4fe0-bac7-2f0b4a8e0fd9'}>
-          {t('about.availability_statement')}
-        </MuiLink>
-        <MuiLink data-testid={dataTestId.footer.privacyLink} component={Link} to={UrlPathTemplate.PrivacyPolicy}>
-          {t('privacy.privacy_statement')}
-        </MuiLink>
-      </Box>
+      <div>
+        <Box sx={{ display: 'flex', gap: '0.5rem', color: 'primary.main' }}>
+          <Typography
+            sx={{
+              fontWeight: 'bold',
+              color: 'primary.main',
+            }}>
+            {t('footer.about_nva')}
+          </Typography>
+
+          <MuiLink
+            sx={{
+              justifySelf: { xs: 'center', lg: 'start' },
+              gridRow: { xs: 3, lg: 1 },
+              marginLeft: 'auto',
+              color: 'primary.main',
+              display: 'flex',
+              gap: '0.5rem',
+            }}
```

Comment:
Ser ut som dette kan forenkles en del, med samme teknikker som vi gjorde tidligere for "hvordan bli kunde". Mye enklere å komme tilbake å gjøre endringer på koden senere jo mindre kode man må ta stilling til
```suggestion
        <Box sx={{ display: 'flex', gap: '0.5rem', justifyContent: 'space-between' }}>
          <Typography sx={{ fontWeight: 'bold', color: 'primary.main' }}>{t('footer.about_nva')}</Typography>

          <MuiLink
            sx={{ color: 'primary.main', display: 'flex', gap: '0.5rem' }}
```